cmd/zed/agents/zfs_mod.c

```diff
@@ -545,9 +562,11 @@ zfs_process_add(zpool_handle_t *zhp, nvlist_t *vdev, boolean_t labeled)
 	 * Wait for udev to verify the links exist, then auto-replace
 	 * the leaf disk at same physical location.
 	 */
-	if (zpool_label_disk_wait(path, 3000) != 0) {
-		zed_log_msg(LOG_WARNING, "zfs_mod: expected replacement "
-		    "disk %s is missing", path);
+	if (zpool_label_disk_wait(path, DISK_LABEL_WAIT) != 0) {
```

Comment:
`DISK_LABEL_WAIT` is 30000, value was previously 3000. I'm pretty sure that was the intent (I recall a private discussion about it) but just pointing it out to be sure.

Reply:
The values is the maximum time to wait, but the call can return as soon as the link is available.  3 seconds is pretty close to a potential latency margins from the udev layer . Where as 30 seconds is what zpool create | add have been using.